.env.production

```
@@ -42,3 +42,5 @@ ALGOLIA_INDEX_NAME=PRODUCTION
 
 # Support
 SUPPORT_EMAIL_ADDRESS=support@passculture.app
+CGU_LINK=https://docs.passculture.app/textes-normatifs/mentions-legales-et-conditions-generales-dutilisation-de-lapplication-pass-culture
+PRIVACY_POLICY=https://docs.passculture.app/textes-normatifs/charte-des-donnees-personnelles
```

Juste pour être consistent avec l'autre variable `CGU_LINK` que tu as ajouté :)
```suggestion
PRIVACY_POLICY_LINK=https://docs.passculture.app/textes-normatifs/charte-des-donnees-personnelles
```